Tests/SecureXPCTests/Client & Server/Error Integration Tests.swift

```diff
@@ -24,7 +24,7 @@ class ErrorIntegrationTests: XCTestCase {
                                    .throwsType(ExampleError.self)
         let server = XPCServer.makeAnonymous()
         let client = XPCClient.forEndpoint(server.endpoint)
-        server.registerRoute(failureRoute) {
+        server.registerRoute(failureRoute) { connectionToken in
```

Comment:
```suggestion
        server.registerRoute(failureRoute) { _ in
```
and the same bellow to prevent Xcode warnings 👀

Reply:
ok, done